html-forms.tar/html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>Email</p>
+    <input type="text" id="comment" name="comment" value="" />
```

Comment:
this should be of `type="email"` since we are collecting an email here, the `id` and `name` attributes should probably be valued like this instead of "comment"

`<input type="email" id="email-address" name="email">`

If you set the email like the example I gave above, the label on line 7 should have a `for` attribute equal to "email" so they are linked.